en/flight_modes/mission.md

```diff
@@ -215,23 +215,25 @@ If the vehicle is already flying when the mission is started, a takeoff mission
 
 Starting flights with mission takeoff (and landing using a mission landing) is the recommended way of operating a plane autonomously.
 
-:::note

```

Comment:
This page updated. You are still expected to go to the Takeoff mode page to understand config of the catapult or runway BUT you don't need to go there to understand how the mission item works.

Reply:
Note, I hope to split this into MC and FW pages as well. Will need help to understand what an FW mission looks like in terms of supported mission items etc.

Reply:
Yes Mission should be split up as well, the behavior is greatly different between MC and FW

Reply:
Do you think you could outline the differences - both what is different in the current doc and "overall"?